src/frontend/src/test.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
We can delete the test files, if we ever need them we can once again restore them from Git

Reply:
yes